session/manager_http.go

```diff
@@ -178,13 +193,16 @@ func (s *ManagerHTTP) extractToken(r *http.Request) string {
 	return token
 }
 
-func (s *ManagerHTTP) FetchFromRequest(ctx context.Context, r *http.Request) (*Session, error) {
+func (s *ManagerHTTP) FetchFromRequest(ctx context.Context, r *http.Request) (_ *Session, err error) {
+	ctx, span := s.r.Tracer(ctx).Tracer().Start(ctx, "sessions.ManagerHTTP.FetchFromRequest")
+	defer otelx.End(span, &err)
+
 	token := s.extractToken(r)
 	if token == "" {
 		return nil, errors.WithStack(NewErrNoCredentialsForSession())
 	}
 
-	se, err := s.r.SessionPersister().GetSessionByToken(ctx, token, ExpandEverything)
+	se, err := s.r.SessionPersister().GetSessionByToken(ctx, token, ExpandEverything, identity.ExpandDefault)
```

Comment:
We can optimize this further here. Session expand doesn't have to be `ExpandEverything` every time. The expand for devices is only ever needed in the `whoami` handler

Reply:
That's true, ideally it would be `ExpandNothing`. But chaning this is a serious breaking change, which I did not want to address API versioning in this PR.

Reply:
I was thinking of a safer - `ExpandIdentity` rather than nothing since the Identity state is also needed to determine if the session may even be used/active

Reply:
Ah true, that makes sense. We could however also optimize the SQL query to return the active state from the identity without needing to expand the identity itself using a JOIN

Reply:
Yeah, makes sense. I'll also create an issue to track the `active` attribute relying on `expires_at` like we discussed earlier

Reply:
👍 would you be open to create such an issue? :)

Reply:
- Use expiry timestamp vs boolean flag GH issue opened - https://github.com/ory-corp/cloud/issues/3895
- Fix Identity expand for session query https://github.com/ory-corp/cloud/issues/3896

@aeneasr